.eslintrc.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+# this is a test. 'use strict';
```

Comment:
test fix

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Don’t use iterators. Prefer JavaScript’s higher-order functions instead of loops like for-in or for-of. eslint: no-iterator no-restricted-syntax 

---
**Sema Reaction:** :question: I have a question | **Sema Tags:** Readable

Reply:
test

Reply:
test

Reply:
one more test

Reply:
test

Reply:
test

Reply:
tetstt a

Reply:
sd sd sa

Reply:
test

Reply:
test test fixgood
The purpose of having consensus and following it is to free up energy, not to consume it in favor of poorly-defined or subjective goals such as "it looks better". Having each one one make up his/her own rules "and damn the rest" is just as antisocial as forcing everyone to adhere to strict guidelines which dictate where you put your commas or spacing. Discipline provides freedom to the group, because once you achieve consensus on a matter, the effort of dealing with other's trespasses is greater than the effort of keeping oneself from trespassing.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Unreadable, Secure, Inefficient, Not maintainable